package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
General comment: The empty module icon is very hard to see in light mode, and effectively invisible in the module select dropdowns. Ideally that icon would flip from white to black, but that'd require a separate sprite in the sheet that doesn't exist. Could we just add this as a special case and add a slightly darker background for that specific icon? If you disagree, maybe I can just do a minor revision after this to add a 'light mode' sprite for that icon.

Reply:
Let me play around with css rules and see what can be done first before trying to add a new sprite for it. 

I think a tint variable for an "empty" icon set would be appropriate that could be applied?

Reply:
In theory that definitely sounds nice, if it can be done without an extra sprite that would be preferred.

Reply:
To do this I added the icon id as a css class to the image tag and a css filter rule that applies in light mode. The approach seems less than ideal on the css class but the filter works nicely.